backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -352,6 +358,14 @@ public void Remove(Parcel parcel)
             this.Context.Parcels.Remove(entity); // TODO: Shouldn't be allowed to permanently delete parcels entirely.
             this.Context.CommitTransaction();
         }
+
+        private static void CanUpdateOrRemoveProperty(string projectNumber, bool isAdmin)
```

Comment:
This again looks like a reasonable extension method.
`CanUpdateOrRemoveProperty(this ClaimsPrincipal user, Parcel parcl) ...`
`CanUpdateOrRemoveProperty(this ClaimsPrincipal user, Building building) ...`